instrumentation/elasticsearch/elasticsearch-transport-5.3/src/test/groovy/Elasticsearch53TransportClientTest.groovy

```diff
@@ -36,257 +37,257 @@ import static io.opentelemetry.trace.Span.Kind.CLIENT
 import static org.elasticsearch.cluster.ClusterName.CLUSTER_NAME_SETTING
 
 class Elasticsearch53TransportClientTest extends AgentTestRunner {
-  public static final long TIMEOUT = 10000; // 10 seconds
+    public static final long TIMEOUT = 10000; // 10 seconds
```

Comment:
can you try reformatting this file in Intellij? i'm not sure why it's not using our `.editorconfig`